tutorials/3d/portals/tutorial_simple.rst

```diff
@@ -0,0 +1,83 @@
+
+Unzip this project and add it to the Project Manager using the "Import" button.
+
+Introduction
+~~~~~~~~~~~~
+This tutorial will introduce you to building a "Hello World" room system with two rooms, and a portal in between.
+
+Step 1
+~~~~~~
+.. image:: tutorial_simple/img/tutorial_simple1.png
+
+- Create a new project
+- Add a :ref:`Spatial<class_Spatial>` as the scene root (I have called it 'Root')
+- Next add a :ref:`RoomManager<class_RoomManager>` node. We will need this later to process the room system.
+- Next we need to start defining our rooms. We create all our rooms under another Spatial we have called 'RoomList'.
+- Add a new :ref:`Room<class_Room>` node as a child of the roomlist.
+- Give it a  name that starts with the prefix ``Room_``. We add our chosen name as a suffix, here we have used 'kitchen'.
+- We will now create the geometry of our room. The names you give to the geometry is up to you.
+- Create a :ref:`MeshInstance<class_MeshInstance>` for the floor. Create a box mesh, and scale and position it to form a floor.
+- Create MeshInstances for the walls. Again use box meshes, and scale and position them, but be sure to leave an opening on one side (you will need to create two wall segments to do this on that side).
+
+Step 2
+~~~~~~
+.. image:: tutorial_simple/img/tutorial_simple2.png
+
+- Now we need to create the other room.
+- You can do this simply by duplicating the first room (select the 'Room_kitchen' node, right click and choose ``duplicate``).
+- Rotate and position the second room so that the openings line up.
+- Rename the second room to 'Room_lounge'.
+
+Step 3
+~~~~~~
+.. image:: tutorial_simple/img/tutorial_simple3.png
+
+- Next we will add a portal between the two rooms
+- Create a new :ref:`Portal<class_Portal>` in the kitchen, and call it ``Portal_lounge``. The naming scheme tells the system which room it should link to.
+- Scale and position the portal using the node ``Transform`` in the inspector, so it fits within the opening between the two rooms.
+- The portal plane should face *outward* from the source room, i.e. towards the lounge. This direction is indicated by the arrow in the editor gizmo, and the color of the portal.
+
+Step 4
+~~~~~~
+.. image:: tutorial_simple/img/tutorial_simple4.png
+
+- In order to make things more exciting, we want to add a few more boxes to the rooms.
+- Placing these boxes as children / grand children of the room nodes explicitly tells the system which room the objects should be in. However, we can also create these objects *outside* the rooms. Provided they are in the ``roomlist`` branch, the system will attempt to automatically place them in the correct room at runtime.
+- I've placed these boxes as children of a Spatial I have called 'Freeform', just to keep our scene tree neat and tidy.
+- I've also used a green material to make the boxes stand out more.
+- Let's also place an :ref:`OmniLight<class_OmniLight>` so it will be autoplaced in one of the rooms.
```

Comment:
```suggestion
- To make things more exciting, we want to add a few more boxes to the rooms.
- Placing these boxes as children or grandchildren of the room nodes explicitly tells the system which room the objects should be in. However, we can also create these objects *outside* the rooms. Provided they are in the RoomList branch, the system will attempt to automatically place them in the correct room at runtime.
- On the screenshot, the boxes were placed as children of a Spatial called ``Freeform`` to keep things tidy.
- Boxes also have a green SpatialMaterial assigned to them to make boxes stand out more from the rest of the room.
- Let's also create an :ref:`OmniLight<class_OmniLight>` so it will be autoplaced in one of the rooms.
```